### PR TITLE
Disabled helpcenter feature and hidden the ui on onboarding

### PR DESCRIPTION
**Pull request by ajayadav09**
## Proposed changes

https://jira.newfold.com/browse/PRESS6-49

- Have disabled the helpcenter feature via the feature class in wp-module-features
- toggled the helpcenter UI

## Type of Change

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [ ] I have viewed my change in a web-browser
- [ ] Linting and tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: includes/HelpCenterFeatureHooks.php [hunk 1]
```diff
@@ -28,6 +29,32 @@ public function __construct() {
...
+	 * Enqueue inline script as back up to hide the helpcenter UI on onboarding flow
+	 */
+	public function clearhelpCenter() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_clear_storage_script' ), 20 );
+	}
+
+	/**
+	 * Inline script to clear localStorage
+	 */
+	public function enqueue_clear_storage_script() {
```
Comment (officiallygod) [inline includes/HelpCenterFeatureHooks.php:45]: Clear storage can be a misleading name ig cause it just changes the class?

File: includes/HelpCenterFeatureHooks.php [hunk 2]
```diff
@@ -37,8 +64,14 @@ public function hooks() {
 	 * @return boolean the filtered value
 	 */
 	public function filterValue( $value ) {
+		$helpCenterFeature = Features::getInstance()->getFeature( 'helpCenter' );
 		if ( $this->shouldDisable() ) {
```
Comment (officiallygod) [inline includes/HelpCenterFeatureHooks.php:68]: Can we combine this check on top? `if ( $helpCenterFeature )`

File: includes/HelpCenterFeatureHooks.php [hunk 3]
```diff
@@ -37,8 +64,14 @@ public function hooks() {
 	 * @return boolean the filtered value
 	 */
 	public function filterValue( $value ) {
+		$helpCenterFeature = Features::getInstance()->getFeature( 'helpCenter' );
```
Comment (officiallygod) [inline includes/HelpCenterFeatureHooks.php:67]: Maybe use the already-defined variable to keep everything consistent?
https://github.com/newfold-labs/wp-module-help-center/blob/a09d1cf6bf5c95bc93d79842f8c46ce3472caef2/includes/HelpCenterFeature.php#L23

File: includes/HelpCenterFeatureHooks.php [hunk 4]
```diff
@@ -28,6 +29,32 @@ public function __construct() {
...
+
+		$clear_storage_js = <<<JS
+			document.addEventListener("DOMContentLoaded", function() {
+				const nfdHelpContainer = document.getElementById( 'nfd-help-center' );
+				nfdHelpContainer.classList.toggle( 'help-container', false );
+			});
+		JS;
+
+		// Add the inline script to `nfd-help-center` if it’s enqueued
+		if ( wp_script_is( 'nfd-help-center', 'enqueued' ) ) {
```
Comment (officiallygod) [inline includes/HelpCenterFeatureHooks.php:55]: Just wanted to check, this always remains enqueued right?
Also, https://github.com/newfold-labs/wp-module-help-center/blob/a09d1cf6bf5c95bc93d79842f8c46ce3472caef2/includes/HelpCenter.php#L26

**Review by kreamweb — APPROVED** [review]
I think that is good!

**Pull request merged**